docs/droneworld/about.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Drone World
+custom_edit_url: https://github.com/oss-slu/DroneWorld
+---
+
+## Overview
+DroneWorld is a platform for testing small unmanned aerial systems (sUAS) applications by simulating realistic test scenarios based on specified requirements.DroneWorld allows users to configure test scenarios by specifying environmental conditions (e.g. weather, terrain), sUAS capabilities (sensors, hardware), and test properties derived from system requirements. It generates a realistic 3D simulation environment using digital shadow models and runs the user's sUAS application within it. DroneWorld monitors and collects data to check if safety properties are violated. It produces test reports analyzing simulation results.
+
+### Information
+
+- **Source Code:**  https://github.com/oss-slu/DroneWorld
+- **Client** Dr.Ankit Agrawal - https://github.com/ankyAgrawal
+- **Current Tech Lead:** Yashaswini Shivalingaiah (https://github.com/yashaswini-slu, https://www.linkedin.com/in/yashaswini-shivalingaiah-467a9652/)
+- **Developers:**
+    - Kaleb Yu (https://github.com/kalyus)
+    - Szymon Rostkowski (https://github.com/sr259)
+    - Hebron Bekele (https://github.com/hebronh)
+- **Start Date:** 25 Aug 2023
```

Comment:
@yashaswini-slu I think the start date is not right - this should be the start date of Dr. Agrawal's drone world project.

Reply:
resolved.